Framework/Algorithms/test/GetEiV1Test.h

```diff
@@ -170,17 +172,15 @@ class GetEiV1Test : public CxxTest::TestSuite {
     const double peakOneCentre(6493.0), sigmaSqOne(150. * 150.),
         peakTwoCentre(10625.), sigmaSqTwo(25. * 25.);
     const double peakOneHeight(3000.), peakTwoHeight(1000.);
-    for (int i = 0; i <= numBins; ++i) {
-      const double xValue = 5.0 + 5.5 * i;
-      if (includePeaks && i < numBins) {
+    for (int i = 0; i < numBins; ++i) {
+      if (includePeaks) {
         testWS->dataY(0)[i] =
             peakOneHeight *
-            exp(-0.5 * pow(xValue - peakOneCentre, 2.) / sigmaSqOne);
+            exp(-0.5 * pow(xdata[i] - peakOneCentre, 2.) / sigmaSqOne);
```

Comment:
@quantumsteve The `int` overloads seem to be removed with c++11. The new variant (7) claims it casts `int` to `double`. If you think there might be a worthwhile speedup, we can rewrite the `pow` as a product.